css/main.css

```diff
@@ -0,0 +1,142 @@
+* {
+	box-sizing: border-box;
+}
+
+
```

Comment:
white space is a great way to make your code readable. But, you should be consistent with how and where you add it.